ecf/include/envir-p1.h

```diff
@@ -2,6 +2,13 @@
 export job=${job:-$PBS_JOBNAME}
 export jobid=${jobid:-$job.$PBS_JOBID}
 
+export RUN_ENVIR=emc
+export envir=%ENVIR%
+export MACHINE_SITE=%MACHINE_SITE%
+
+if [ -n "%SENDCANNEDDBN:%" ]; then export SENDCANNEDDBN=${SENDCANNEDDBN:-%SENDCANNEDDBN:%}; fi
```

Comment:
I don't understand this;
`if [ -n "%SENDCANNEDDBN:%" ]`
The `-n` checks if the string variable length is non-zero.  If it is true, then
`export SENDCANNEDDBN=${SENDCANNEDDBN:-%SENDCANNEDDBN:%};`
If it is already non-zero, should this not already be in the environment space?
What am I missing?

Reply:
This is for develop be able to overwrite in case of the need to test. This was used for NCO testing.